src/coreservices.h

```diff
@@ -37,10 +39,6 @@ class CoreServices : public QObject {
     CoreServices(const CmdlineArgs& args);
     ~CoreServices() = default;
 
-    void initializeSettings();
-    // FIXME: should be private, but WMainMenuBar needs it initialized early
```

Comment:
I am confused why you removed this comment.

Reply:
The comment says that `initializeKeyboard()` should be private. This commit makes it private. Hence, the comment is not needed anymore.

Reply:
How does this not break WMainMenuBar? If you remove a FIXME comment like this, please explain why the problem that the comment documented is resolved in the commit message.

Reply:
The comment says it needs to be initialized early. The commit message states that the settings and keyboard are now initialized in the CoreServices constructor, that's as early as possible without pulling them out of CoreServices. Please elaborate what is unclear here.

Reply:
Sorry for the noise. I took a closer look at the code and your explanation makes sense. In retrospect I don't know why I didn't make this method private and call it in the constructor in the first place.